ios/RNInstabug/InstabugReactBridge.m

```diff
@@ -382,6 +382,39 @@ - (dispatch_queue_t)methodQueue {
     [Instabug clearAllExperiments];
 }
 
+RCT_EXPORT_METHOD(addFeatureFlags:(NSDictionary *)featureFlagsMap) {
+    NSMutableArray<IBGFeatureFlag *> *featureFlags = [NSMutableArray array];
+    for(id key in featureFlagsMap){
+        NSString* variant =((NSString * )[featureFlagsMap objectForKey:key]);
+        if ([variant length]==0) {
+            [featureFlags addObject:[[IBGFeatureFlag alloc] initWithName:key]];
+        }
+        else{
+            [featureFlags addObject:[[IBGFeatureFlag alloc] initWithName:key variant:variant]];
+
+        }
+    }
+
+    [Instabug addFeatureFlags:featureFlags];
+}
+
+RCT_EXPORT_METHOD(removeFeatureFlags:(NSArray *)featureFlags) {
+    NSMutableArray<IBGFeatureFlag *> *features = [NSMutableArray array];
+    for(id item in featureFlags){
+            [features addObject:[[IBGFeatureFlag alloc] initWithName:item]];
+        }
+
+    @try {
+    [Instabug removeFeatureFlags:features];
+   } @catch (NSException *exception) {
+            NSLog(@"%@", exception);
+     }
+}
```

Comment:
No need for the conversion to `NSMutableArray<IBGFeatureFlag *>*`, we can just pass the `NSArray *` as is to `removFeatureFlags`.
```suggestion
RCT_EXPORT_METHOD(removeFeatureFlags:(NSArray *)featureFlags) {
    @try {
        [Instabug removeFeatureFlags:featureFlags];
    } @catch (NSException *exception) {
        NSLog(@"IBG-RN: %@", exception);
    }
}
```